slides/linear_model.md

```diff
@@ -61,18 +60,20 @@ e.g. salary should be explain by education level (nb of year of study)
 ---
 # Linear regression
 A linear model is a slope "as close as possible" from datapoint
-The blue curve is the prediction for each *x*
+The blue curve is the prediction for each **x**
 
 .shift-down.pull-left.shift-left[<img src="../figures/linear_fit.svg" width="100%">]
-
+.pull-right.very-small[`From sklearn.linear_model import LinearRegression`
```

Comment:
```suggestion
.pull-right.very-small[`from sklearn.linear_model import LinearRegression`
```
Python is case sensitive.

Also, it there a way to do code blocks / syntax highlighting in markdown? We should probably use it here

Reply:
I'll look for syntax highlighting in markdown yes.